src/js/components/message-indicator.jsx

```diff
@@ -0,0 +1,81 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import DocumentTitle from 'react-document-title';
+
+const BLINKING_INTERVAL = 1500;
+
+export class MessageIndicatorComponent extends Component {
+    state = {
+        initialDocumentTitle: global.document ? document.title : '',
```

Comment:
Good point, but I wonder how we can effectively track that. I mean, what if the page change occurs while the message is shown... I guess the user's app will update the title when changing url...?